pennylane/labs/dla/dense_util.py

```diff
@@ -234,6 +234,169 @@ def batched_pauli_decompose(H: TensorLike, tol: Optional[float] = None, pauli: b
     return H_ops
 
 
+def check_commutation(ops1, ops2, vspace):
+    r"""Helper function to check :math:`[\text{ops1}, \text{ops2}] \subseteq \text{vspace}`
+
+    .. warning:: This function is expensive to compute
```

Comment:
💡

Reply:
should I add this symbol here 😅 ?

Reply:
No, I just meant that it was a good idea to point it out explicitly XD